src/hooks/useSubscriptions.ts

```diff
@@ -84,9 +83,7 @@ export function useProposalsSubscription(filters: FiltersType) {
       onProposalsHandler
     )
 
-    return () => {
-      proposalsSubscription.current = null
-    }
+    return () => proposalsSubscription.current.unsubscribe()
```

Comment:
This should be:

```suggestion
    return () => {
      proposalsSubscription.current.unsubscribe()
    }
```
because useEffect expects undefined during unsubscribing.